Artiom/src/main/java/Retailer.java

```diff
@@ -0,0 +1,65 @@
+
+import java.util.*;
+
+public class Retailer {
+    public static void main(String[] args) {
+
+        //Create first order
+        Map<String, Address> firstUserAddress = new HashMap<>();
+        firstUserAddress.put("Home", new Address("Moldova", "Chisinau", "Stefan cel Mare"));
+        User firstUser = new User("Pikachu", firstUserAddress);
+        List<Item> firstItemList = new ArrayList<>();
+        firstItemList.add(new Item("iPhone", Type.GADGET));
+        Order firstOrder = new Order(firstUser, firstItemList);
+
+        //Create second order
+        Map<String, Address> secondUserAddress = new HashMap<>();
+        secondUserAddress.put("Home", new Address("Ukraine", "Kyiv", "Verkhnia Street"));
+        User secondUser = new User("Slowpoke", secondUserAddress);
+        List<Item> secondItemList = new ArrayList<>();
+        Order secondOrder = new Order(secondUser, secondItemList);
+
+        //Create third order
+        Map<String, Address> thirdUserAddress = new HashMap<>();
+        thirdUserAddress.put("Work", new Address("Romania", "Bucharest", "Rope Street"));
+        User thirdUser = new User("Bulbasaur", thirdUserAddress);
+        List<Item> thirdListItem = new ArrayList<>();
+        Order thirdOrder = new Order(thirdUser, thirdListItem);
+
+        //Create list of orders
+        List<Order> orders = new ArrayList<>();
+        orders.add(firstOrder);
+        orders.add(secondOrder);
+        orders.add(thirdOrder);
+
+        //Create list of Users
+        List<User> users = new ArrayList<>();
+        users.add(firstUser);
+        users.add(secondUser);
+        users.add(thirdUser);
+
+    }
+
+    //Create method to filter orders by CLOTHES type
+    public static List<Order> filterOrdersByType(List<Order> orders, Type type) {
+        List<Order> filteredOrders = new ArrayList<>();
+        for (Order filteredOrder : orders) {
+            if (orders.contains(Type.CLOTHES)) {
```

Comment:
You are iterating over orders list:
`for (Order filteredOrder : orders) {...}`
So you should use filteredOrder in your if condition. Also you will need one more method in Order class that will filter your single order and let you know if there is specific Type in this order.

Reply:
done